project/MicroServiceBuild.scala

```diff
@@ -26,15 +26,15 @@ private object AppDependencies {
   import play.sbt.PlayImport._
   import play.core.PlayVersion
 
-  private val microserviceBootstrap = "6.18.0"
-  private val playReactivemongoVersion = "5.2.0"
+  private val microserviceBootstrap = "10.2.0"
```

Comment:
There is an update to this version
```suggestion
  private val microserviceBootstrap = "10.3.0"
```